packages/flutter_tools/test/general.shard/windows/visual_studio_test.dart

```diff
@@ -40,9 +40,9 @@ void main() {
     'isRebootRequired': false,
     'isComplete': true,
     'isLaunchable': true,
+    'isPrerelease': false,
     'catalog': <String, dynamic>{
       'productDisplayVersion': '15.9.12',
-      'productMilestoneIsPreRelease': true,
     }
   };
 
```

Comment:
Please add a variant of this (_oldResponse?) that doesn't contain the reboot/complete/launchable entries, and at least one test using them; `installationHasIssues` still has asserts that the keys are present; we should have a test coverage that would catch that. (And also, please fix that 😀)

Reply:
Done